### PR TITLE
Nazar/feature/add new users

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: app/controllers/user.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from functools import wraps
+from flask import abort, redirect, request, url_for
+from flask_login import current_user
+
+
+from app.logger import log
+
+
+def role_required(required_role: list[str]):
```
Comment (nixissmtp) [inline app/controllers/user.py:9]: could we use list[Enum]?

File: app/views/auth.py [hunk 2]
```diff
@@ -90,11 +91,12 @@ def login():
         if current_user.role == m.UsersRole.admin:
             log(log.INFO, "Redirecting to users page.")
             return redirect(url_for("user.get_all"))
-        else:
+        elif current_user.role == m.UsersRole.dealer:
```
Comment (nixissmtp) [inline app/views/auth.py:94]: m.UserRole.dealer.value maybe?

File: app/views/label_location.py [hunk 3]
```diff
@@ -1,62 +1,69 @@
...
 
 location_blueprint = Blueprint(
     "labels/locations", __name__, url_prefix="/labels/locations"
 )
 
 
-@location_blueprint.route("/create", methods=["GET", "POST"])
+@location_blueprint.route("/create", methods=["GET"])
+@login_required
+@role_required([m.UsersRole.admin.value, m.UsersRole.dealer.value])
```
Comment (nixissmtp) [inline app/views/label_location.py:24]: try to put login_required in role_required decorator

File: app/views/sellers.py [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+    if not form.validate_on_submit():
+        log(log.ERROR, "Invalid form data [%s]", form.errors)
+        flash(f"Invalid form data [{form.format_errors}]", "danger")
+        return redirect(url_for("user.seller.get_user_sellers"))
+
+    user = m.User(
+        first_name=form.first_name.data,
+        last_name=form.last_name.data,
+        email=form.email.data,
+        role=m.UsersRole.seller,
```
Comment (nixissmtp) [inline app/views/sellers.py:90]: seller.value

**Review by nixissmtp — CHANGES_REQUESTED** [review]

File: app/views/sellers.py (hunk 4 above)
Comment (Nazar-kob) [reply app/views/sellers.py:90]: can be like this too

File: app/views/auth.py (hunk 2 above)
Comment (Nazar-kob) [reply app/views/auth.py:94]: thanks

File: app/controllers/user.py (hunk 1 above)
Comment (Nazar-kob) [reply app/controllers/user.py]: thanks

**Pull request merged**